doc/start/hardware-recommendations.rst

```diff
@@ -120,16 +121,17 @@ multiple daemons against a single drive can slow performance considerably.
 Hard Disk Drives
 ----------------
 
-OSDs should have plenty of hard disk drive space for object data. We recommend a
-minimum hard disk drive size of 1 terabyte. Consider the cost-per-gigabyte
+OSDs should have plenty of hard disk drive space for object data. We recommend
+a minimum hard disk drive size of 1 terabyte. Consider the cost-per-gigabyte
```

Comment:
Suggest that `hard disk` in lines 124-125 is redundant, remove from the above two lines.

Reply:
With fresh eyes, I agree. It seems to me that the original use of "hard disk" was intended to distinguish HDD from SSD, which I suppose nowadays is not unlike distinguishing blunt stick from pointy stick.